cmd/cosign/cli/options/sign.go

```diff
@@ -137,7 +137,7 @@ func (o *SignOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&o.IssueCertificate, "issue-certificate", false,
 		"issue a code signing certificate from Fulcio, even if a key is provided")
 
-	cmd.Flags().StringVar(&o.SignContainerIdentity, "sign-container-identity", "",
+	cmd.Flags().StringArrayVar(&o.SignContainerIdentities, "sign-container-identity", nil,
```

Comment:
I'd use `StringSliceVar` here so you can specify the values as:

`--sign-container-identity=docker.io/name1:tag1,docker.io/name2:tag2`

Reply:
Thanks, updated.